python/cuml/internals/api_context_managers.py

```diff
@@ -117,6 +164,11 @@ def set_api_output_dtype(output_dtype):
 
 class InternalAPIContext(contextlib.ExitStack):
     def __init__(self):
+        """
+        Constructor for InternalAPIContext class.
```

Comment:
It should be obvious that this is a constructor. Instead, just document what this context manager is. Why is it necessary? When is it used?